pages/playerprofile.tsx

```diff
@@ -0,0 +1,56 @@
+import { useState } from "react";
```

Comment:
let's rename this file to playerProfile to maintain consistency with the rest of the times we reference it